ansible/utils/cli_test.py

```diff
@@ -123,7 +123,7 @@ def test_cli(client, folder, opts):
         if item['name'].startswith('TCGA-02'):
             testItem = item
             break
-    localFile = client.listFile(testItem['_id']).next()
+    localFile = next(client.listFile(testItem['_id']))
```

Comment:
The py2/3 safe way to call this would be `six.next(...)`

Reply:
I thought `next` worked the same in Python 2 and Python 3 provided Python >= 2.6.

Reply:
I have encountered situations in the past where `next` did not work but replacing with `six.next` did, but those may have been weird edge cases of iterators or something. If this works, we can go with it.